pages/common/idnits.md

```diff
@@ -0,0 +1,25 @@
+# idnits
+
+> Check internet-drafts for submission nits.
+> Looks for violations of Section 2.1 and 2.2 of the requirements listed on <https://www.ietf.org/id-info/checklist>.
+> More information: <https://tools.ietf.org/tools/idnits/>.
+
+- Check a file for nits:
+
+`idnits {{path/to/file.txt}}`
```

Comment:
I don't know what a `nit` is, but can they also occur in other file formats?

Reply:
Looking at [(a fork of) the source](https://github.com/fno2010/idnits/blob/d38815d5330d80571ba030a6033f35686375c482/idnits), it seems like it prefers `.txt` files, although I believe it will take `.html` files, and _possibly_ `.(txt|pdf|xml|doc|nr)` 🤔

Reply:
Yeah, it's uncommon for whatever reason.  But the IETF definitely still uses txts mostly, from the 60s 😆

Reply:
The oldest rfc, [rfc1](https://datatracker.ietf.org/doc/html/rfc1), is from 1969 👀

>7 April 1969

Reply:
I'm always surprised by how long computers actually exist 😅

Reply:
Yeah. Off the top of my head, the oldest version of Fortran is from '55 or '58, I think...

Reply:
Wow, that's old.